contrib/hadoop/src/main/java/org/xtreemfs/common/clients/hadoop/XtreemFSFileSystem.java

```diff
@@ -84,26 +84,14 @@ public void initialize(URI uri, Configuration conf) throws IOException {
             Logging.logMessage(Logging.LEVEL_DEBUG, this, "init : " + uri);
         }
 
-        String defaultURIString = conf.get("fs.defaultFS");
```

Comment:
yes I agree too. Fallback code is a good idea. never thought about mixing sources before.